sci-libs/itk/itk-4.8.1.ebuild

```diff
@@ -0,0 +1,151 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
Yes, `EAPI=6` can even be stable'd nowadays. This might require changing some cmake-utils uses, but we generally would people to only submit `EAPI=6` from now on.